src/main/java/fact/extraction/SinglePulseExtraction.java

```diff
@@ -8,7 +8,8 @@
 import stream.annotations.Parameter;
 import java.util.Arrays;
 import java.util.ArrayList;
-import fact.features.singlePulse.timeLineExtraction.SinglePulseExtractor;
+import fact.features.singlePulse.timeSeriesExtraction.SinglePulseExtractor;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
```

Comment:
maybe we want move`fact.features.singlePulse.timeSeriesExtraction.ElementWise` to `fact.Utils` or a library folder with helper classes that we put there.